src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php

```diff
@@ -57,14 +57,12 @@ public function create(string $resourceClass, string $property, array $options =
         foreach ($identifiers as $identifier) {
             if ($identifier === $property) {
                 $propertyMetadata = $propertyMetadata->withIdentifier(true);
+                $writable = false;
```

Comment:
Yes but here it is assigned 2 times instead of 1 in most cases (`$doctrineClassMetadata` is almost always an instance of `ClassMetadataInfo`): first it is assigned to false, then it is instantly assigned to the value of `$doctrineClassMetadata->isIdentifierNatural()`).